nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-restapi-processors/src/main/java/org/apache/nifi/processors/elasticsearch/PutElasticsearchJson.java

```diff
@@ -95,21 +101,28 @@
         .name("put-es-json-error-documents")
         .displayName("Output Error Documents")
         .description("If this configuration property is true, the response from Elasticsearch will be examined for failed documents " +
-                "and the failed documents will be sent to the \"errors\" relationship.")
+                "and the failed documents will be sent to the \"" + REL_FAILED_DOCUMENTS.getName() + "\" relationship.")
         .allowableValues("true", "false")
         .defaultValue("false")
         .expressionLanguageSupported(ExpressionLanguageScope.NONE)
         .required(true)
         .build();
 
-    static final Relationship REL_FAILED_DOCUMENTS = new Relationship.Builder()
-            .name("errors").description("If \"" + OUTPUT_ERROR_DOCUMENTS.getDisplayName() + "\" is set, " +
-                    "any FlowFile that failed to process the way it was configured will be sent to this relationship " +
-                    "as part of a failed document set.")
-            .autoTerminateDefault(true).build();
+    static final PropertyDescriptor NOT_FOUND_IS_SUCCESSFUL = new PropertyDescriptor.Builder()
+        .name("put-es-json-not_found-is-error")
+        .displayName("Treat \"Not Found\" as Error")
+        .description("If true, \"not_found\" Elasticsearch Documents will be routed to the \"" + REL_SUCCESS.getName() +
+                "\" (true) or the \"" + REL_FAILED_DOCUMENTS.getName() + "\" (false) relationship.")
```

Comment:
Since you already say "if true", I think the (true)/(false) are redundant.  Could be: "If true, 'not_found' Elasticsearch Documents will be routed to the 'success' relationship, otherwise they will be routed to the 'errors' relationship"